src/kubectl-util.test.ts

```diff
@@ -163,4 +163,18 @@ describe('Testing all functions in kubectl-util file.', () => {
       })
       expect(toolCache.find).toHaveBeenCalledWith('kubectl', 'v2.0.0')
    })
+
+   test('getKubectlPath() - installs kubectl if version is provided but not cached', async () => {
+      jest.restoreAllMocks()
```

Comment:
let's move the restoreAllMocks() to an [afterEach](https://jestjs.io/docs/setup-teardown#repeating-setup)